lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -707,16 +707,27 @@ sub sdaf_execute_remover {
         '--type', $type_parameter,
         '--auto-approve');
 
-    # capture command output into log file
-    my $output_log_file = log_dir() . "/cleanup_$args{deployment_type}.txt";
-    $remover_cmd = log_command_output(command => $remover_cmd, log_file => $output_log_file);
-
-    # SDAF must be executed from the profile directory, otherwise it will fail
-    assert_script_run("cd " . $tfvars_path);
-    record_info('SDAF destroy', "Executing SDAF remover:\n$remover_cmd");
-    # Keep the timeout high, definitely above 1H. Azure tends to be slow.
-    my $rc = script_run($remover_cmd, timeout => 7200);
-    upload_logs($output_log_file, log_name => $output_log_file);
+    my $rc;
```

Comment:
This code could have unit testing in file os-autoinst-distri-opensuse/t/20_sdaf_deployment_library.t

Reply:
Added simple Unit Test.